recipes/muler/meta.yaml

```diff
@@ -0,0 +1,57 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "muler" %}
+{% set version = "0.2.1" %}
```

Comment:
Latest is now 0.2.4. It'd be good to go ahead and updated unless you specifically need 0.2.1.